### PR TITLE
add json logging example

### PR DESCRIPTION
**Pull request by zeitlinger**
@cyrille-leclerc the code already works - automated tests are stell missing

### CODE REVIEW

File: examples/java/json-logging/k8s/collector-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+data:
+  otel-collector-config.yaml: |- 
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:4317
+          http:
+            endpoint: 0.0.0.0:4318
+      filelog/json-elastic:
```
Comment (cyrille-leclerc) [inline examples/java/json-logging/k8s/collector-configmap.yaml]:
```suggestion
      filelog/json-ecs:
```

File: examples/java/json-logging/k8s/collector-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+          exporters: [ otlphttp/traces ]
+        metrics:
+          receivers: [ otlp ]
+          processors: [ batch ]
+          exporters: [ otlphttp/metrics ]
+        logs/otlp:
+          receivers: [ otlp ]
+          processors: [ batch ]
+          exporters: [ otlphttp/logs ]
+        logs/json-elastic:
```
Comment (cyrille-leclerc) [inline examples/java/json-logging/k8s/collector-configmap.yaml]:
```suggestion
        logs/json-ecs:
```

File: examples/java/json-logging/k8s/collector-configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+          receivers: [ filelog/json-elastic ]
```
Comment (cyrille-leclerc) [inline examples/java/json-logging/k8s/collector-configmap.yaml]:
```suggestion
          receivers: [ filelog/json-ecs ]
```

File: examples/java/json-logging/k8s/dice.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        - containerPort: 8080
+        env:
+          - name: OTEL_EXPORTER_OTLP_ENDPOINT
+            value: "http://lgtm:4318"
+          - name: OTEL_LOGS_EXPORTER
+            value: "none" # to avoid duplicate logs
+          - name: OTEL_RESOURCE_ATTRIBUTES
+            value: service.name=dice,service.namespace=shop,service.version=1.1,deployment.environment=staging
+          - name: OTEL_INSTRUMENTATION_COMMON_MDC_RESOURCE_ATTRIBUTES
+            value: "service.namespace,service.instance.id,deployment.environment"
```
Comment (cyrille-leclerc) [inline examples/java/json-logging/k8s/dice.yaml]: Do we also output `service.version`?

File: examples/java/json-logging/logback.xml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<configuration>
```
Comment (zeitlinger) [inline examples/java/json-logging/logback.xml]: use spring-logback.xml

File: examples/java/json-logging/README.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# Exporting Application logs using JSON logging in Kubernetes
+
+## Introduction
+
+todo
```
Comment (zeitlinger) [inline examples/java/json-logging/README.md]: copy steps from other pr

- see https://www.elastic.co/guide/en/ecs-logging/java/current/setup.html 
- link to new grafana page

File: .github/workflows/acceptance-tests.yml [hunk 7]
```diff
@@ -12,7 +12,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: grafana/oats
-          ref: d07befda3cfa162865d8081aa64501ea26578870
+          ref: 214c70a70ed78ef273db52dc75555536476ce15f
```
Comment (zeitlinger) [inline .github/workflows/acceptance-tests.yml]: from https://github.com/grafana/oats/pull/46

**Pull request merged**